numpy/ma/extras.py

```diff
@@ -730,6 +730,10 @@ def compress_rowcols(x, axis=None):
 
     Parameters
     ----------
+    x : array_like, MaskedArray
+        The array to mask.  If not a MaskedArray instance (or if no array
+        elements are masked).  The result is a MaskedArray with `mask` set
```

Comment:
I think that should not be a period, but a comma, i.e. `If not a MaskedArray instance (or if no array elements are masked, the result is a MaskedArray with...`